src/utils/helpers/chat.ts

```diff
@@ -62,3 +62,14 @@ export const templateTime = (hour: number, minute: number) => {
 
   return `오전 ${String(hour).padStart(2, '0')}:${minuteStartWithZero}`;
 };
+
+export const getSpecificUser = (memberList: Member[], userId: number) => {
+  const userData = memberList.filter((member) => member.userId === userId);
```

Comment:
p5: find를 사용해보는 것에 대해선 어떻게 생각하시나요!? getSpecificUser라는 이름에서 미루어 보았을 때 1명만 찾으면 될 것 같고, 그러한 메서드로 적절한 건 Array로 타입 추론이 되는 filter보다 하나의 객체로 타입 추론이 되는 find가 더 적절할 것 같아서요! 일치하는 첫 번째 요소만 필요한 거라면 find도 고려해보시면 좋지 않을까 싶습니다!😀

```suggestion
  const foundUser = memberList.find(member => member.userId === userId);
```

Reply:
와 아니 제가 찾던겁니다 ㅋㅋㅋㅋㅋ이거 분명 다른 메서드있는데 뭐지??하다가 filter썼는데 추천 감사합니다 👍

Reply:
반영했습니다 [dcde1ee](https://github.com/prgrms-web-devcourse/Team-DarkNight-Kkini-FE/pull/224/commits/dcde1ee630ac60416c3fffe7bfda6c1e2ae7e5f3)